float64 angle_0
float64 angle_1
float64 angle_2
---
bool success
